packages/neuron-wallet/src/controllers/app/index.ts

```diff
@@ -83,6 +87,9 @@ export default class AppController {
       connectionStatus,
       codeHash,
     }
+
+    AppController.handleWalletListChange()
```

Comment:
`getInitState` would be called multiple times, which makes the subscription repeated.

Reply:
Where's the appropriate place to initialize this?

Reply:
We can subscribe to the subjects just on the app launched.

Reply:
I see. Thanks.

Reply:
@Keith-CY Moved that to `app.ready` event block. Please let me know if there's a better launch entry.